.cloudbees/workflows/workflow.yaml

```diff
@@ -8,6 +8,9 @@ on:
       - '**'
 jobs:
   build:
```

Comment:
Do you want to add the `if` condition(s) @stephenc added to his action workflows? For the `build` job he added this condition - https://github.com/cloudbees-io/checkout/blob/main/.cloudbees/workflows/workflow.yml#L13 and for the `test` job he added this condition - https://github.com/cloudbees-io/checkout/blob/main/.cloudbees/workflows/workflow.yml#L50